lib/src/chunk_builder.dart

```diff
@@ -93,6 +93,16 @@ class ChunkBuilder {
   /// block.
   bool _firstFlushLeft = false;
 
+  /// Splitting is completely disabled inside string interpolation. We do still
```

Comment:
[nit] blank line after the first sentence - maybe pick a different first sentence too?

Reply:
Done.